src/cascadia/LocalTests_SettingsModel/SerializationTests.cpp

```diff
@@ -399,75 +391,71 @@ namespace SettingsModelLocalTests
-                                                    ]
-                                                },
-                                                "schemes": [
-                                                    {
-                                                        "name": "Cinnamon Roll",
-
-                                                        "cursorColor": "#FFFFFD",
-                                                        "selectionBackground": "#FFFFFF",
-
-                                                        "background": "#3C0315",
-                                                        "foreground": "#FFFFFD",
-
-                                                        "black": "#282A2E",
-                                                        "blue": "#0170C5",
-                                                        "cyan": "#3F8D83",
-                                                        "green": "#76AB23",
-                                                        "purple": "#7D498F",
-                                                        "red": "#BD0940",
-                                                        "white": "#FFFFFD",
-                                                        "yellow": "#E0DE48",
-                                                        "brightBlack": "#676E7A",
-                                                        "brightBlue": "#5C98C5",
-                                                        "brightCyan": "#8ABEB7",
-                                                        "brightGreen": "#B5D680",
-                                                        "brightPurple": "#AC79BB",
-                                                        "brightRed": "#BD6D85",
-                                                        "brightWhite": "#FFFFFD",
-                                                        "brightYellow": "#FFFD76"
-                                                    }
-                                                ],
-                                                "actions": [
-                                                    { "command": { "action": "renameTab", "title": "Liang Tab" }, "keys": "ctrl+t" },
-                                                    { "command": { "action": "sendInput", "input": "VT Griese Mode" }, "keys": "ctrl+k" },
-                                                    { "command": { "action": "renameWindow", "name": "Hecker Window" }, "keys": "ctrl+l" }
-                                                ]
-                                            })" };
-
-        auto settings{ winrt::make_self<implementation::CascadiaSettings>(false) };
-        settings->_ParseJsonString(settingsString, false);
-        settings->_ApplyDefaultsFromUserSettings();
-        settings->LayerJson(settings->_userSettings);
-        settings->_ValidateSettings();
+            "$help" : "https://aka.ms/terminal-documentation",

```

Comment:
Since I removed the helpful comments from `userDefaults.json`, I thought a documentation link might be useful for some. It's the first field in the `settings.json`.